We need to export the CRYPTOPP_DISABLE_ASM flag if cryptopp was not built
with SIMD support. Certain functions prototypes are guarded by this
flag and build failures will arise in other ports if not set appropriately.

--- GNUmakefile.orig	2018-04-08 08:47:12 UTC
+++ GNUmakefile
@@ -90,7 +90,7 @@ SUNCC_512_OR_LATER := $(shell echo "$(SUNCC_VERSION)" 
 SUNCC_513_OR_LATER := $(shell echo "$(SUNCC_VERSION)" | $(GREP) -i -c -E "CC: (Sun|Studio) .* (5\.1[3-9]|5\.[2-9]|6\.)")
 
 # Enable shared object versioning for Linux
-HAS_SOLIB_VERSION := $(IS_LINUX)
+HAS_SOLIB_VERSION := 1
 
 # Fixup SunOS
 ifeq ($(IS_SUN),1)
@@ -957,7 +957,7 @@ libcryptopp.pc:
 	@echo 'Version: 7.0' >> libcryptopp.pc
 	@echo 'URL: https://cryptopp.com/' >> libcryptopp.pc
 	@echo '' >> libcryptopp.pc
-	@echo 'Cflags: -I$${includedir}' >> libcryptopp.pc
+	@echo 'Cflags: -I$${includedir} $(findstring -DCRYPTOPP_DISABLE_ASM,$(CXXFLAGS))' >> libcryptopp.pc
 	@echo 'Libs: -L$${libdir} -lcryptopp' >> libcryptopp.pc
 
 # This recipe prepares the distro files
